examples/go-samples/spewer/run.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/hashicorp/go-hclog"
+	"github.com/lyraproj/servicesdk/lang/go/lyra"
+)
+
+type spewIn struct {
+	It interface{}
```

Comment:
One idea could be to use a px.Value here and then use `px.ToPrettyString(in.It)`, perhaps in combination with `spew.Sdump`, or at different log levels.

Reply:
changed